go.mod

```diff
@@ -74,7 +74,7 @@ require (
 	github.com/google/btree v1.1.2
 	github.com/google/go-attestation v0.4.4-0.20220404204839-8820d49b18d9
 	github.com/google/go-cmp v0.5.9
-	github.com/google/go-tpm-tools v0.3.11
+	github.com/google/go-tpm-tools v0.3.12
```

Comment:
FYI @strideynet. I didn't see anything too striking, but letting you know anyway.

Reply:
Taking a look now

Reply:
Looks fine !